src/domain/event/eventForm/EventForm.tsx

```diff
@@ -101,22 +105,48 @@ const EventForm = () => {
 
               <h2>{t('eventForm.location.title')}</h2>
 
-              {/* TODO: Add location selector component here when implemented */}
+              <FormGroup>
+                <Field
+                  helperText={t('eventForm.location.helperTextPlace')}
+                  labelText={t('eventForm.location.labelPlace')}
+                  name="place"
+                  component={PlaceSelectorField}
+                />
+              </FormGroup>
+              {!!place && (
+                <FormGroup>
+                  <PlaceInfo id={place} />
+                </FormGroup>
+              )}
+              <FormGroup>
+                <Field
```

Comment:
I haven't used Formik much but just curious: are the many benefits using the render prop component instead of useFormik hook?

Reply:
I think useFormik is meant for advanced use cases e.g people would not like to store form state to React Store. https://jaredpalmer.com/formik/docs/api/useFormik

Reply:
I was thinking this because useFormik is used first in the turotial. But later it says:

"To save you even more time, Formik comes with React Context-powered API/component make life easier and less verbose: <Formik />, <Form />, <Field />, and <ErrorMessage />. More explicitly, they use React Context implicitly to connect to the parent <Formik /> state/methods."

So yeah seems like the <Formik /> is the way to go